### DISCUSSION TITLE
Regex match query

### DISCUSSION THREAD

**Discussion by nbupadhya** [mockoon/mockoon#947] [Help]
How to make regex match on a body parameter by making the value case insensitive? 
User6, user6, uSER6 all these should match. Added image for reference. 

Using V1.22 on ubuntu (V 20 ) desktop

<img width="894" alt="image" src="https://user-images.githubusercontent.com/22615750/218960959-922bcff1-f3f0-42d8-b967-d274aff13340.png">

**Comment by 255kb** [comment]
Currently, regex check is case sensitive. There is an open issue #586 to add a case insensitive option.

**Reply by nbupadhya** [reply]
Thanks for the reply, will keep the discussion open for now
